### ISSUE TITLE
Obtaining SBPs for real IMU data later on

### ISSUE THREAD

**Issue by TKojar** [jyf588/transformer-inertial-poser#20]
Hi, thank you for your work. I am implementing your methodology for the upper body (in particular three IMUs on hand, forearm and upper arm, three IMUs on shoulder midback and pelvis). 

For now, I see that you have provided us with the SBPs. But later on if we want to train using actual patient data, how would we go generate those SBPs? You mentioned 

>Note: we did two additional things to the DIP dataset: first as DIP do not have root motion, we use a pre-trained (trained without DIP) model to label pseudo ground-truth SBPs for the DIP motions. In this repo we omit this tedious step and directly provide the pre-trained SBP info to append to the DIP dataset

so I am wondering if you can at least sketch out your process. Thank you.
